### PR TITLE
DATAUP-494: Remove inline css/styling from job status table; move to SCSS files

### PR DESCRIPTION
**Pull request by ialarmedalien**
# Description of PR purpose/changes

Removing that grotesque inline style and moving it to the stylesheet, where it belongs.

# Jira Ticket / Issue #
e.g. https://kbase-jira.atlassian.net/browse/DATAUP-494
- [x] Added the Jira Ticket to the title of the PR (e.g. `DATAUP-69 Adds a PR template`)

# Testing Instructions
* Details for how to test the PR:
- [x] Tests pass locally and in GitHub Actions
- [x] Changes available by spinning up a local narrative and navigating to see that the job status table is now styled using css!

# Dev Checklist:

- [x] My code follows the guidelines at https://sites.google.com/lbl.gov/trussresources/home?authuser=0
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] (JavaScript) I have run Prettier and ESLint on changed code manually or with a git precommit hook

### CODE REVIEW

File: kbase-extension/scss/partials/_jobStatus.scss [hunk 1]
```diff
@@ -82,23 +71,28 @@
   }
 
   &__cell {
-    &--status {
-      text-transform: capitalize;
```
Comment (ialarmedalien) [inline kbase-extension/scss/partials/_jobStatus.scss:86]: this wasn't used

File: kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js [hunk 2]
```diff
@@ -203,13 +199,14 @@ define([
             dataTable = $(container)
                 .find('table')
                 .dataTable({
+                    autoWidth: false,
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js:202]: the magical declaration!

File: nbextensions/bulkImportCell/bulkImportCell.js [hunk 3]
```diff
@@ -602,16 +602,17 @@ define([
             setupMessageBus();
             setupDomNode();
 
+            // finalize by updating the lastLoaded attribute, which triggers a toolbar re-render
+            const meta = cell.metadata;
+            meta.kbase.attributes.lastLoaded = new Date().toUTCString();
+            cell.metadata = meta;
+
```
Comment (ialarmedalien) [inline nbextensions/bulkImportCell/bulkImportCell.js:609]: moving the job manager init down

File: kbase-extension/scss/partials/_bulkImportCell.scss [hunk 4]
```diff
@@ -63,7 +63,6 @@ $tab-widget-padding: 4px;
   // kb-bulk-import__tab_pane_widget
   &__tab_pane_widget {
     margin: $tab-widget-margin;
-    padding: $tab-widget-padding;
```
Comment (ialarmedalien) [inline kbase-extension/scss/partials/_bulkImportCell.scss:66]: causing the tab pane to overflow its box

**Review by briehl — APPROVED** [review]
👍

**Pull request merged**